gallery/src/pages/Accessibility.tsx

```diff
@@ -0,0 +1,72 @@
+/*
+ * Copyright (c) 2019-present Sonatype, Inc.
+ * This program and the accompanying materials are made available under
+ * the terms of the Eclipse Public License 2.0 which accompanies this
+ * distribution and is available at https://www.eclipse.org/legal/epl-2.0/.
+ */
+import React from 'react';
+import {GalleryTile} from '../gallery-components/GalleryTiles';
+import { NxP, NxCode, NxH3, NxList, NxTextLink } from '@sonatype/react-shared-components';
+
+const Accessibility = () =>
+  <GalleryTile title="Accessibility">
+    <NxP>
+      Accessibility (A11y) means ensuring that our applications are usable by everybody. We do this in a variety of
+      different ways. We make sure that our designs are clear and that they follow accessibility guidelines for
+      contrast and readability. Our components are written with semantic HTML, correct use of <NxCode>aria</NxCode>
+      tags, and testing in VoiceOver and ChromeVox accessibility is built into the Shared Components. However there is
+      still work that needs to be done to make sure that the application as a whole is accessible.
+    </NxP>
+
+    <NxH3 className="nx-tile__section-header">Making your pages accessible</NxH3>
+    <NxP>
+      Ensuring you use the components in an accessible manner is important, here are a few tips to help you make your
+      pages accessible.
+    </NxP>
+
+    <NxList className="nx-list--bulleted">
```

Comment:
Reading this section makes me realizes that our bulleted lists don't follow the max text width.  I suspect the text contents in our normal lists don't either.  We should put in a bug about that

Reply:
> Reading this section makes me realizes that our bulleted lists don't follow the max text width. I suspect the text contents in our normal lists don't either. We should put in a bug about that

https://issues.sonatype.org/browse/RSC-654